### ISSUE TITLE
trachea cyst and GI cyst

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4836]
**Preferred term label:**
trachea cyst

**Synonyms**
n/a

**Definition (free text, please give PubMed ID)**
Tracheal cyst is a rare entity and occurs from evagination of embryologic tracheal buds, which may occur anywhere along the trachea These cysts usually contains air and may be completely asymptomatic or may become infected. 
http://medind.nic.in/ibq/t05/i3/ibqt05i3p224.pdf

**Parent term (use hpo.jax.org/app)**
HP_0002778 'Abnormal trachea morphology' or
HP_0032445 'Pulmonary cyst' 
?

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

For Kids First dataset, the original description is:
trachea cyst with respiratory-type epithelium

**Comment by nicolevasilevsky** [comment]
**Preferred term label:**
gastrointestinal cyst

**Synonyms**
GI cyst

**Definition (free text, please give PubMed ID)**
Cystic lesions of the gastrointestinal (GI) tract rarely occur and can be divided into several categories: congenital lesions, neoplastic lesions, and miscellaneous lesions. Most of the cystic masses of the GI tract are discovered incidentally and usually appear as submucosal lesions.
PMID:20592930

**Parent term (use hpo.jax.org/app)**
 'Morphological abnormality of the gastrointestinal tract'

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

For Kids First dataset, the original description is:
similar cystic structure in GI tissue

**Comment by pnrobinson** [comment]
Is this the same as Tracheal duplication cyst ?
https://academic.oup.com/jcem/article/100/1/19/2812698

**Comment by nicolevasilevsky** [comment]
I will see what i can find out

**Comment by pnrobinson** [comment]
I am adding this as such, I think it is the only kind of cyst seen. Let's reopen if there is more information.
